### PR TITLE
[release 4.6] Bug 1889676: Gather installplans

### PR DESCRIPTION
**Pull request by martinkunc**
Based on cherry-picks for installplans only from #192 
Adding Gathering of installplans

### CODE REVIEW

**Review by tisnik — APPROVED** [review]
LGTM

File: pkg/gather/clusterconfig/clusterconfig.go [hunk 1]
```diff
@@ -725,22 +734,35 @@ func GatherClusterProxy(i *Gatherer) func() ([]record.Record, []error) {
...
+			for _, sr := range csrs {
+				records = append(records, record.Record{Name: fmt.Sprintf("config/certificatesigningrequests/%s", sr.ObjectMeta.Name), Item: sr})
+			}
+
+			// If the Continue field is not set, this should be the end of available data.
+			// Otherwise, update the Continue value and perform another request iteration.
+			if requests.Continue == "" {
+				break
+			}
+			continueValue = requests.Continue
```
Comment (tremes) [inline pkg/gather/clusterconfig/clusterconfig.go:765]: I guess this change shouldn't be included here.

**Comment by 0sewa0** [comment]
closing it, inf favor of https://github.com/openshift/insights-operator/pull/243

**Pull request closed without merging**